src/notation/view/notationpaintview.cpp

```diff
@@ -691,6 +691,25 @@ void NotationPaintView::hoverMoveEvent(QHoverEvent* event)
 void NotationPaintView::keyReleaseEvent(QKeyEvent* event)
 {
     if (isInited()) {
+        std::string sequence = QKeySequence(event->key()).toString().toStdString();
+        if (event->key() < Qt::Key_Shift && sequence.length() > 0) {
+            if (event->modifiers() & Qt::KeyboardModifier::ShiftModifier) {
+                sequence = "Shift+" + sequence;
+            }
+            if (event->modifiers() & Qt::KeyboardModifier::AltModifier) {
+                sequence = "Alt+" + sequence;
+            }
+            if (event->modifiers() & Qt::KeyboardModifier::ControlModifier) {
+                sequence = "Ctrl+" + sequence;
+            }
+            const auto& shortcuts = shortcutsRegister()->shortcutsForSequence(sequence);
+            const std::shared_ptr<IUiActionsRegister> theRegister = actionsRegister();
+            if (std::find_if(shortcuts.begin(), shortcuts.end(), [theRegister](auto s) {
+                return theRegister->actionState(s.action).enabled;
+            }) != shortcuts.end()) {
+                return;
```

Comment:
Basically need to ensure that any shortcut key that there is an enabled handler for is not passed through to the text edit element.  I honestly don't know how this worked in MU3.

Reply:
It's not very clear why this code is needed?
Do you want to check that the action is enabled?
This is done in `void ShortcutsController::activate (const std::string& sequence)` for all shortcuts in the same way